internal/testutil/testutil.go

```diff
@@ -0,0 +1,42 @@
+package testutil
+
+import (
+	"embed"
+)
+
+//go:embed data
+var testDataDir embed.FS
+
+// Name of the root folder in embed.FS
+const testDataRootDir = "data"
+
+// Version represents a kubernetes version. Mock servers can be instantiated to return known output for a given version.
+type Version string
+
+// Server returns an HTTP Server for the given version, ready to serve static endpoints for KSM, Kubelet and CP components.
+func (v Version) Server() (*Server, error) {
+	return newServer(v)
+}
+
+// List of all the versions we have testdata for.
+// When adding a new version:
+// - REMEMBER TO ADD IT TO AllVersions() BELOW.
+// - UPDATE LatestVersion() BELOW IF NEEDED
+
+const (
+	Testdata116 = "1_16"
+	Testdata118 = "1_18"
+)
+
+// LatestVersion returns the latest version we have test data for.
+func LatestVersion() Version {
+	return Testdata118
+}
+
+// AllVersions returns a list of versions we have test data for.
+func AllVersions() []Version {
+	return []Version{
+		Testdata116,
```

Comment:
should we add a comment saying to which KSM version corresponds to each of these?

Reply:
Yep, this is something I though about. My plan is to overwrite all the static files so all of them use the KSM we support, but in the future we might want to have weirder versions.

Reply:
I will add a comment explaining this, and in the future, when I write the script to generate these files, I will keep in midn to log the version used somewhere.